scaffoldmaker/utils/eft_utils.py

```diff
@@ -98,8 +98,8 @@ def remapEftNodeValueLabel(eft, localNodeIndexes, fromValueLabel, expressionTerm
     :param expressionTerms: List of (valueLabel, scaleFactorIndexesList ) to remap to.
         e.g. [ (Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS3, [5, 6]) ]
     '''
-    functionCount = eft.getNumberOfFunctions()
-    for f in range(1, functionCount + 1):
+    functionCount = eft.getNumberOfFunctions()    
```

Comment:
Remove whitespace changes in this and following line. This file should not be modified.